README.md

```diff
@@ -248,6 +248,39 @@ module.exports = {
 }
 ```
 
+### Locales
+
+Often we render multiple versions of our application for different locations, eg. Australia & New Zealand. To render an HTML file for each location you can use the locales option in `sku.config.js`. Locales are preferable to [Monorepos](monorepo-support) when you need to render multiple versions of your HTML file but only need one version of each of the assets (JS, CSS, images, etc). Note: You can use `locales` inside a monorepo project.
```

Comment:
Sorry—I think `monorepo-support` needs to be `#monorepo-support` for the link to work correctly. Also, "Monorepos" should be lowercase.